.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Release
+about: Announce and track an upcoming release
+labels:
+  - release
+---
+
+|  |  |
+| --- | --- |
+| Release version | YYYY.MM.X |
+| Planned release date | YYYY-MM-DD |
+| Status | On-track |
+| Release manager | @github_handle |
+
+## Best effort
+
+_Try to close before the release but will not block the release_
+
+- [ ] ...
+- [ ] ...
+
+## Blocker
+
+_Issues that would cause us to block and postpone the release if not fixed_
+
+- [ ] ...
+- [ ] ...
+
+## Comments
+
+...
```

Comment:
What would go in this section?

Reply:
Just wanted to leave a place for jotting down anything else that's relevant. For example, I often cc you, Ben, Julia, and Florian and ask about any known blockers. That's one thing I anticipate going in this section

Reply:
Maybe it is worth having a stakeholders section?

Reply:
I'm personally okay with just a free-form comments section to start. Happy to change in the future if we think a dedicated stakeholders section would be useful for the group